sda/internal/database/db_functions.go

```diff
@@ -65,30 +65,35 @@ func (dbs *SDAdb) getFileID(corrID string) (string, error) {
 
 // GetInboxFilePathFromID checks if a file exists in the database for a given user and fileID
 // and that is not yet archived
-func (dbs *SDAdb) GetInboxFilePathFromID(submissionUser, fileID string) (string, error) {
+func (dbs *SDAdb) GetInboxFilePathFromID(submissionUser, fileID string, excludeIngested bool) (string, error) {
 	var (
 		err      error
 		count    int
 		filePath string
 	)
 
 	for count == 0 || (err != nil && count < RetryTimes) {
-		filePath, err = dbs.getInboxFilePathFromID(submissionUser, fileID)
+		filePath, err = dbs.getInboxFilePathFromID(submissionUser, fileID, excludeIngested)
 		count++
 	}
 
 	return filePath, err
 }
 
-func (dbs *SDAdb) getInboxFilePathFromID(submissionUser, fileID string) (string, error) {
+func (dbs *SDAdb) getInboxFilePathFromID(submissionUser, fileID string, excludeIngested bool) (string, error) {
 	dbs.checkAndReconnectIfNeeded()
 	db := dbs.DB
 
-	const getFilePath = "SELECT submission_file_path from sda.files where " +
+	var getFilePath = "SELECT submission_file_path from sda.files where " +
```

Comment:
```suggestion
	const getFilePath = "SELECT submission_file_path from sda.files where " +
```

DB calls should never be variables in order to prevent SQL injection issues.

Reply:
removed the commit